Source/WebCore/platform/graphics/cg/GraphicsContextCG.cpp

```diff
@@ -1087,25 +1086,39 @@ static void setCGStyle(CGContextRef context, const std::optional<GraphicsStyle>&
 
     auto cgStyle = WTF::switchOn(*style,
         [&] (const GraphicsDropShadow& dropShadow) -> RetainPtr<CGStyleRef> {
+#if HAVE(CGSTYLE_CREATE_SHADOW2)
             return adoptCF(CGStyleCreateShadow2(dropShadow.offset, dropShadow.radius.width(), cachedCGColor(dropShadow.color).get()));
+#else
+            UNUSED_PARAM(dropShadow);
```

Comment:
Should we assert that we never get here?

Reply:
Done.